### DISCUSSION TITLE
如何监听评论加载事件呢？

### DISCUSSION THREAD

**Discussion by inkss** [walinejs/waline#715] [Q&A] [answered]
首次评论加载完成/点击更多/新增评论时等。

我想为评论中的图片增加灯箱支持，需要相对应的事件以方便处理评论内容。

**Comment by lizheming** [comment, marked as answer]
灯箱可以参见文档 https://waline.js.org/advanced/faq.html#%E5%A6%82%E4%BD%95%E5%A2%9E%E5%8A%A0%E7%81%AF%E7%AE%B1%E6%95%88%E6%9E%9C 本质是使用了事件委托

**Reply by inkss** [reply]
好的

**Reply by inkss** [reply]
不过这种没办法使用分组了

**Reply by lizheming** [reply]
没办法使用分组是什么意思？

**Reply by inkss** [reply]
这个的 [grouping](https://fancyapps.com/docs/ui/fancybox/#grouping)，不能归类到一起

**Reply by lizheming** [reply]
https://fancyapps.com/docs/ui/fancybox/api 自己看下 Fancybox 的 API，有个 `groupAll` 配置哈。
